server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java

```diff
@@ -835,4 +838,146 @@ public void testExpireAt() {
+      assertThat(redis.sort("numbers", SortArgs.Builder.by("nosort").limit(0, 1)))
+            .containsExactly( "1");
+      // SORT numbers DESC BY nosort
+      assertThat(redis.sort("numbers", SortArgs.Builder.by("nosort").desc()))
+            .containsExactly( "3", "-22", "19", "-1", "0", "1", "8", "4", "3", "1");
+      // SORT numbers DESC BY nosort LIMIT 0 1
+      assertThat(redis.sort("numbers", SortArgs.Builder.by("nosort").limit(0, 1).desc()))
+            .containsExactly( "3");
+      // SORT numbers BY something
+      assertThat(redis.sort("numbers", SortArgs.Builder.by("something")))
+            .containsExactly( "1", "3", "4", "8", "1", "0", "-1", "19", "-22", "3");
+      // SORT numbers BY w_*
+      assertThat(redis.sort("numbers", SortArgs.Builder.by("w_*")))
+            .containsExactly( "-1", "-22", "0", "1", "1", "19", "3", "3", "4", "8");
+      // RPUSH people ryan tristan pedro
+      redis.rpush("people", "ryan", "tristan", "pedro", "michael");
+      // SORT people BY w_*
+      assertThat(redis.sort("people", SortArgs.Builder.by("w_*")))
+            .containsExactly( "michael", "pedro", "ryan", "tristan");
+      // SET w_ryan 1
+      redis.set("w_ryan", "1");
+      // SET w_pedro 2
+      redis.set("w_pedro", "2");
+      // SET w_tristan 3
+      redis.set("w_tristan", "3");
+      // SET w_michael 4
+      redis.set("w_michael", "4");
+      // SORT people BY w_*
+      assertThat(redis.sort("people", SortArgs.Builder.by("w_*")))
+            .containsExactly( "ryan", "pedro", "tristan", "michael");
+      // SET o_ryan 1
+      redis.set("o_ryan", "persistence");
+      // SET o_pedro 2
+      redis.set("o_pedro", "cross-site");
+      // SET o_tristan 3
+      redis.set("o_tristan", "security");
+      // SORT people BY w_* GET o_*
+      assertThat(redis.sort("people", SortArgs.Builder.by("w_*").get("o_*")))
+            .containsExactly( "persistence", "cross-site", "security", null);
+      // SORT people BY w_* GET o_* GET #
+      assertThat(redis.sort("people", SortArgs.Builder.by("w_*").get("o_*").get("#")))
+            .containsExactly( "persistence", "ryan", "cross-site", "pedro",
+                  "security", "tristan", null, "michael");
+      // SORT people BY w_* GET o GET #
+      assertThat(redis.sort("people", SortArgs.Builder.by("w_*").get("o").get("#")))
+            .containsExactly( null, "ryan", null, "pedro",
+                  null, "tristan", null, "michael");
+
+      // *****
+      // Sets
+      // *****
+      // SADD set_numbers 1 3 4 8 1 0 -1 19 -22 3
+      redis.sadd("set_numbers", "1", "3", "4", "8", "1", "0", "-1", "19", "-22", "3");
+      // SORT numbers
+      assertThat(redis.sort("set_numbers"))
+            .containsExactly("-22", "-1", "0", "1", "3", "4", "8", "19");
+      // SORT set_numbers ALPHA
+      assertThat(redis.sort("set_numbers", SortArgs.Builder.alpha()))
+            .containsExactly("-1", "-22", "0", "1", "19", "3", "4", "8");
```

Comment:
same as for sort list

Reply:
could you please update to redis_version:7.2.0 ?